src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,37 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({
+  post: {
+    title,
+    user,
+    body,
+    comments,
+  },
+}) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {title}
+      </h3>
+
+      <UserInfo user={user} key={user.id} />
```

Comment:
We need `key` only when we render a list of similar components/elements (when we use `map`). So key here is redundant